.run/EmployeesdepApplication.run.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Remove this file from repository by updating .gitignore file

Reply:
I added a new String in .gitignore. Don't know wheather it's enougth

Reply:
I don't find, how to delete file from repository selbst, if it's already there